lib/matplotlib/patheffects.py

```diff
@@ -373,3 +375,170 @@ def draw_path(self, renderer, gc, tpath, affine, rgbFace):
         if clip_path:
             self.patch.set_clip_path(*clip_path)
         self.patch.draw(renderer)
+
+
+class TickedStroke(AbstractPathEffect):
+    """
+    A line based PathEffect which draws a path with a ticked style.
+    This line style is frequently used to represent constraints in
+    optimization.  The ticks indicate that one side of the line is
+    invalid.
+
+    The spacing, length, and angle of ticks can be controlled.
+
+    This line style is sometimes referred to as a hatched line, but
```

Comment:
I'd cut the rest of the sentence from 'but in matplotlib...' on.

Reply:
Done